include/common/common.hh

```diff
@@ -12,6 +12,7 @@ using RegId = std::size_t;
 
 constexpr RegId kRegNum = 32;
 constexpr std::uint8_t kBitsInByte = 8;
+constexpr std::uint8_t kXLENInBytes = 4;
```

Comment:
```suggestion
constexpr std::uint8_t kXLENInBytes = sizeof(Word);
```
Because `Word` is a type for storing an instruction

Reply:
done